### ISSUE TITLE
Allow users to add new contacts who are then mailed through the site

### ISSUE THREAD

**Issue by TomSteinberg** [mysociety/fixmytransport#189]
Need something to deal with the following use case from user feedback:

"Last December, during the Xmas snow emergency, bus services listed on the web were not actually running. So I used the early version of FMT to fire off a report.

What I got back was: "Thank you for your email regarding the bus service 50 not operating as per the timetable. The bus service 50 is operated commercially by NXWM and Diamond Buses without a financial subsidy from Centro. Therefore
you will need to contact them direct on the reliability and punctuality of the service 50."

In short, the CustSvc fellow from Centro was saying 'not our problem'. From that point, I reckoned there wasn't much use in reporting further discrepanacies between timetables and realities. "
